cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/ConsentWebView.java

```diff
@@ -39,6 +42,7 @@
 abstract public class ConsentWebView extends WebView {
 
     private static final String TAG = "ConsentWebView";
+    private ConnectivityManager manager;
```

Comment:
here as well we should rename it to connectivityManager

Reply:
@leodoin going forward I will follow connectivityManager as name, However I was following same naming convention from existing code where it was named as "manager"